bash_completionsV2.go

```diff
@@ -177,40 +177,28 @@ __%[1]s_handle_standard_completion_case() {
     local tab=$'\t' comp
 
     local longest=0
+    local compline
     # Look for the longest completion so that we can format things nicely
-    while IFS='' read -r comp; do
+    while IFS='' read -r compline; do
         # Strip any description before checking the length
-        comp=${comp%%%%$tab*}
+        comp=${compline%%$tab*}
         # Only consider the completions that match
         comp=$(compgen -W "$comp" -- "$cur")
```

Comment:
Doing this for every candidate separately would be something to look into getting rid of next, but I suggest we postpone looking into it as far as this PR is concerned and not get delayed because of it here.